test/unit/transform/epacems_test.py

```diff
@@ -37,3 +40,86 @@ def test_harmonize_eia_epa_orispl():
     )
     actual_df = epacems.harmonize_eia_epa_orispl(cems_test_df, crosswalk_test_df)
     pd.testing.assert_frame_equal(expected_df, actual_df, check_dtype=False)
+
+
+def test_epacamd_eia_subplant_ids():
+    """Ensure subplant_id gets applied appropriately to example plants."""
```

Comment:
I think my one remaining question is about where this lil unit test should live! rn there isn't really a place to unit test the `pudl.etl` directory. there is a 👶🏻 glue unit test module but that is running one test from the `pudl.glue` directory.... I kept this her for rn but it feels wrong. @bendnorman  do you have a sense of if/when we should migrate some of these unit tests into a new dagster-ified directory structure? should i leave this here for now? make a new module or dir?

Reply:
Hmm I'm not sure how we want to structure the tests in the long run. For now I feel like a `test/unit/transform/glue_test.py` file would be a good place for this test.